Lombiq.Tests.UI.Samples/Tests/DataBaseSnapshotTests.cs

```diff
@@ -0,0 +1,42 @@
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Samples.Constants;
+using Lombiq.Tests.UI.Services;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// We can execute test on already existing databases. To test this, we will take a snapshot of the running application,
+// and use that.
+public class DataBaseSnapshotTests : UITestBase
+{
+    public DataBaseSnapshotTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Here, we set up the application, then we take a snapshot of it, then we use the
+    // "ExecuteTestFromExistingDBAsync()" to run the test on that. Then we test the basic Orchard features to check that
+    // the application was set up correctly.
+    [Theory, Chrome]
+    public Task BasicOrchardFeaturesShouldWorkWithExistingDBSetup(Browser browser) =>
+         ExecuteTestAsync(
+                async context =>
+                {
+                    const string AppForDataBaseTestFolder = "AppForDataBaseTest";
+
+                    await context.GoToSetupPageAndSetupOrchardCoreAsync(RecipeIds.BasicOrchardFeaturesTests);
+                    await context.Application.TakeSnapshotAsync(AppForDataBaseTestFolder);
+
+                    await ExecuteTestFromExistingDBAsync(
+                         async context => await context.TestBasicOrchardFeaturesExceptSetupAsync(),
+                         browser,
+                         Directory.GetCurrentDirectory() + "//" + AppForDataBaseTestFolder);
```

Comment:
Use the const here too, it should be (made) usable just as for taking the snapshot.

Reply:
Sorry, I don't understand this one, which const do you mean?

Reply:
`AppForDataBaseTestFolder`

Reply:
But I'm already using it, or should it be modified in some way?
![image](https://user-images.githubusercontent.com/80963259/166950701-ff9981d1-f8cd-44a2-978c-69351db27220.png)

Reply:
I mean, use only the const, without `Directory.GetCurrentDirectory() + "//"`: `TakeSnapshotAsync` should work in the same as `ExecuteTestFromExistingDBAsync`.